docs/tags

```diff
@@ -0,0 +1,521 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```

Comment:
Can we remove this file?

I'm not sure where does this come from. I guess it's from your env and you might need to add it in the global .gitignore